var/spack/repos/builtin/packages/mumps/spectrum-mpi-xl.patch

```diff
@@ -0,0 +1,75 @@
+diff -Naur MUMPS_5.0.1/Makefile MUMPS_5.0.1-patched/MUMPS_5.0.1/Makefile
+--- ./Makefile	2015-07-23 13:08:29.000000000 -0400
++++ ./Makefile	2016-12-05 14:08:30.788638382 -0500
+@@ -62,7 +62,7 @@
+ $(libdir)/libpord$(PLAT)$(LIBEXT):
+ 	if [ "$(LPORDDIR)" != "" ] ; then \
+ 	  cd $(LPORDDIR); \
+-	  $(MAKE) CC="$(CC)" CFLAGS="$(OPTC)" AR="$(AR)" RANLIB="$(RANLIB)" OUTC="$(OUTC)" LIBEXT=$(LIBEXT); \
++	  $(MAKE) CC="$(CC)" CFLAGS="$(OPTC)" SAR="$(SAR)" RANLIB="$(RANLIB)" OUTC="$(OUTC)" LIBEXT=$(LIBEXT); \
+ 	fi;
+ 	if [ "$(LPORDDIR)" != "" ] ; then \
+ 	  cp $(LPORDDIR)/libpord$(LIBEXT) $@; \
+diff -Naur MUMPS_5.0.1/PORD/lib/Makefile MUMPS_5.0.1-patched/PORD/lib/MUMPS_5.0.1/Makefile
+--- ./PORD/lib/Makefile	2015-07-23 13:08:29.000000000 -0400
++++ ./PORD/lib/Makefile	2016-12-05 11:26:24.785317467 -0500
+@@ -25,7 +25,7 @@
+ 	$(CC) $(COPTIONS) -c $*.c $(OUTC)$*.o
+ 
+ libpord$(LIBEXT):$(OBJS)
+-	$(AR)$@ $(OBJS)
++	$(SAR)$@ $(OBJS)
+ 	$(RANLIB) $@
+ 
+ clean:
+diff -Naur MUMPS_5.0.1/examples/Makefile MUMPS_5.0.1-patched/examples/Makefile
+--- ./examples/Makefile	2015-07-23 13:08:32.000000000 -0400
++++ ./examples/Makefile	2016-12-05 14:36:10.692857906 -0500
+@@ -25,37 +25,30 @@
+ 
+ LIBSMUMPS = $(libdir)/libsmumps$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
+ 
+-ssimpletest:  $(LIBSMUMPS)  $$@.o
+-	$(FL) -o $@ $(OPTL) ssimpletest.o  $(LIBSMUMPS) $(LORDERINGS) $(LIBS) $(LIBBLAS) $(LIBOTHERS)
```

Comment:
Instead of compiling and linking in 2 different steps using the XL Fortran compiler, I am using a single step to do that.  The XL Fortran compiler does not like to link multiple objects.  I raised this issue w/the compiler folks and until there is a solution, I'll built the binary in a single step.